pygmo/_algorithm_test.py

```diff
@@ -421,3 +423,151 @@ def run_pickle_tests(self):
         self.assertEqual(repr(a), repr(a_))
         self.assertTrue(a.is_(mbh))
         self.assertTrue(a.extract(mbh).inner_algorithm.is_(_algo))
+
+    def run_scipy_wrapper_tests(self):
+        from . import (
+            ackley,
+            algorithm,
+            golomb_ruler,
+            hock_schittkowsky_71,
+            luksan_vlcek1,
+            minlp_rastrigin,
+            population,
+            problem,
+            rastrigin,
+            rosenbrock,
+            scipy,
+        )
+
+        # testing invalid method
+        self.assertRaises(ValueError, lambda: scipy(method="foo"))
+
+        # simple test with ackley, a problem without gradients or constraints
+        methods = ["L-BFGS-B", "TNC", "SLSQP", None]
+        prob = problem(ackley(10))
+        pop = population(prob=prob, size=1, seed=0)
+        init = pop.champion_f
+
+        for m in methods:
+            popc = pop.__copy__()
```

Comment:
Can you use the ``deepcopy`` function from the ``copy`` module, instead of invoking the method?

```python
from copy import deepcopy
popc = deepcopy(pop)
```

Same in the other occurrences below.

Reply:
Fixed in af2f2b9